Common/DocxFormat/Source/DocxFormat/Docx.cpp

```diff
@@ -67,7 +67,7 @@ namespace OOX {
 	void CDocx::FixAfterRead()
 	{
 		//solve id conflict between comments and documentComments
-		if(NULL != m_pComments && m_pComments->m_arrComments.size() > 0 && NULL != m_pDocumentComments && NULL != m_pDocumentComments->m_arrComments.size() > 0)
+		if(NULL != m_pComments && m_pComments->m_arrComments.size() > 0 && NULL != m_pDocumentComments && m_pDocumentComments->m_arrComments.size() > 0)

```

Comment:
maybe: 
`NULL != m_pDocumentComments && m_pDocumentComments->m_arrComments.size() > 0)`

Reply:
This is exactly what is done with that patch

Reply:
yes sorry